src/applications/personalization/profile/mocks/endpoints/user/index.js

```diff
@@ -936,20 +936,7 @@ const baseUserResponses = {
           gender: 'M',
           givenNames: ['Mitchell', 'G'],
           isCernerPatient: false,
-          facilities: [
-            {
-              facilityId: '989',
-              isCerner: false,
-            },
-            {
-              facilityId: '987',
-              isCerner: false,
-            },
-            {
-              facilityId: '983',
-              isCerner: false,
-            },
-          ],
+          facilities: [],
```

Comment:
would the removal of the facilities be worth creating a separate user object for the mock server. Maybe a 'loa3UserWithNoFacilities' or something like that?

Reply:
yes I can do that